apis/core/lib/domain/queries/csv-source.ts

```diff
@@ -5,17 +5,15 @@ import { schema } from '@tpluscode/rdf-ns-builders'
 import { client } from './query-client'
 
 export async function sourceWithFilenameExists(csvMapping: NamedNode, fileName: string): Promise<boolean> {
-  const result = await ASK`
+  return await ASK`
```

Comment:
```suggestion
  return ASK`
```

since it's not wrapped in try/catch, we can catch this on the caller side if need be